lib/Xud.ts

```diff
@@ -183,6 +184,12 @@ class Xud extends EventEmitter {
       // wait for components to initialize in parallel
       await Promise.all(initPromises);
 
+      // We initialize Connext separately because it
+      // requires a NodeKey.
+      await this.swapClientManager.initConnext(
```

Comment:
I originally had this logic in the unlock logic of `InitService`, but then I realized that for the `noencrypt=true` mode that logic is not being used.

Reply:
Created an issue for this: https://github.com/ExchangeUnion/xud/issues/1494 - definitely want this done correctly rather than having a TODO in the code.